package.json

```diff
@@ -44,14 +44,15 @@
     "jspdf": "2.3.1",
     "lattice": "~0.89.2",
     "lattice-auth": "0.22.0",
-    "lattice-sagas": "~0.63.0",
-    "lattice-ui-kit": "~0.43.1",
-    "lattice-utils": "~0.10.0",
+    "lattice-sagas": "~0.66.0",
```

Comment:
Caution - `lattice` and `lattice-sagas` go hand in hand, so it's a bit dangerous to bump only `lattice-sagas`. What are you looking for in `v0.66.0`? Any reason not to bump `lattice` as well?

Reply:
yeah, I'm looking for anything new in `lattice-sagas`, `lattice`, or `lattice-utils` just taking this as an opportunity to bump them. this was an oversight.

Reply:
```suggestion
    "lattice-sagas": "~0.67.3",
```